src/lib/openapi/spec/project-overview-schema.ts

```diff
@@ -135,6 +135,41 @@ export const projectOverviewSchema = {
             description:
                 '`true` if the project was favorited, otherwise `false`.',
         },
+        onboardingStatus: {
+            type: 'object',
+            oneOf: [
+                {
+                    type: 'object',
+                    properties: {
+                        status: {
+                            type: 'string',
+                            enum: ['onboarding-started', 'onboarded'],
```

Comment:
wondering how orval will handle it :)

Reply:
We have similar for lifecycle schemas, should not be too big of an issue. Hopefully